### PR TITLE
Add grpc-stubs via grayskull

### PR DESCRIPTION
**Pull request by asford**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by asford** [comment]
@conda-forge/help-python This is ready for review.

File: recipes/grpc-stubs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - python >=3.6
+  run:
+    - grpcio
+    - mypy >=0.730
+    - python >=3.6
+    - typing-extensions
+
+test:
+  imports:
+    - grpc-stubs
```
Comment (dopplershift) [inline recipes/grpc-stubs/meta.yaml:30]: These aren't valid Python imports, and I'm pretty sure that's not what's actually installed by this package. Might be good to see what [another package](https://github.com/conda-forge/pyspark-stubs-feedstock/blob/b028bc25f0532be7613a7b35fa0256e27a5aa9d8/recipe/meta.yaml#L26-L29) does to test the installation of type stubs.

Comment (asford) [reply recipes/grpc-stubs/meta.yaml:30]: Fantastic, this is exactly what I needed. I believe this test has revealed an error in the upstream distribution, so I'll convert this to a WIP while we investigate upstream.

Comment (asford) [reply recipes/grpc-stubs/meta.yaml]: I've investigated, the upstream lack of `py.typed` definitions is not a blocker, but tracked at https://github.com/shabbyrobe/grpc-stubs/issues/18

Updated with a workaround test.

**Comment by asford** [comment]
@conda-forge/help-python This is ready for re-review.

**Comment by asford** [comment]
@conda-forge/help-python @conda-forge/staged-recipes This is ready for re-review.

**Pull request merged**